src/site/views/orgcommitteedetails/view.html.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+// No direct access
+defined( '_JEXEC' ) or die;
+
+jimport( 'joomla.application.component.view' );
+
+class SwaViewOrgCommitteeDetails extends JViewLegacy {
+    protected $state;
+    protected $item;
+    protected $form;
+    protected $params;
+    protected $user;
+    
+    /**
+     * Display the view
+     */
+    public function display( $tpl = null ) {
+        $app = JFactory::getApplication();
+                
+        $this->user = JFactory::getUser();
+        $this->state = $this->get('State');
+        $this->form = $this->get('Form');
+        $this->params = $app->getParams('com_swa');
+        
+        // Check for errors.
+	if ( count( $errors = $this->get( 'Errors' ) ) ) {
+            throw new Exception( implode( "\n", $errors ) );
+        }
+        
+        // If not logged in
+	if( $this->user->id === 0 ) {
+            $url = 'index.php?option=com_users';
+            $url.= '&return=' . base64_encode( JURI::getInstance()->toString() );
+            $app->redirect( JRoute::_( $url, false ) );
+	}
+        
```

Comment:
Need to make sure the user is a committie member before showing the view.

You can see an example of getting a member object here https://github.com/SWAuk/com_swa/blob/master/src/site/views/universityeventattendees/view.html.php#L39
And checking if they are on a club committie here (although we would want to check org_committee)
https://github.com/SWAuk/com_swa/blob/master/src/site/views/universityeventattendees/view.html.php#L43

This is using the getMember method which is in the base Model class used.
Can be seen at https://github.com/SWAuk/com_swa/blob/orgcommitteedetails/src/site/ModelForm.php#L13 ! :)
The model for orgcommitteedetails is already using this base class so it should all work!